bsip-0060.md

```diff
@@ -0,0 +1,338 @@
+    BSIP: 0060
+    Title: BitShares URI scheme
+    Authors: John Titor, sschiessl, abitmore
```

Comment:
Sorry, I missed this earlier: Please change Authors to "John Titor, Stefan Schießl, Abit More" to match the README.md file. Thanks